meta/documents/user_guide_en.md

```diff
@@ -120,7 +120,7 @@ The following table lists details for settings, format settings and recommended
             Merchant_ProductNumber
         </td>
         <td>
-            <b>Required</b><br>>
+            <b>Required</b><br>
             <b>Limitation:</b> only Numbers allowed, max. 20 characters<br>
             <b>Content:</b> The variation id of the variation.
```

Comment:
Helena has forbid us to set there a "Content:" or in german "Inhalt:". Can you please delete that?